docs/default.nix

```diff
@@ -55,8 +55,6 @@ pkgs.stdenvNoCC.mkDerivation {
 
   patchPhase = ''
     cp ${../README.md} src/README.md
-    cp ${../gnome.png} src/gnome.png
-    cp ${../kde.png} src/kde.png
```

Comment:
As a workaround to keep support for viewing documentation offline, we could fetch the images here using `fetchurl` and copy them into the package. This would also require patching the Markdown files to reference the local image.

- This requires more maintenance effort.
- This still relies on the GitHub links being accessible, at least at build time.